viewer/db.js

```diff
@@ -1666,10 +1682,17 @@ Db.sid2Index = function (id, options) {
   const colon = id.indexOf(':');
 
   if (id[1] === '@') {
-    if (colon > 0) {
-      return 'sessions' + id[0] + '-' + id.substr(2, colon - 2);
+    if (id[0] === 'x') {
+      // ver is x@, which indicates user-specified queryExtraIndices,
+      //   so the id will be formatted x@_index:_id
+      console.log(`Db.sid2Index: ${id.substr(2, colon - 2)}`);
+      return id.substr(2, colon - 2);
+    } else {
```

Comment:
In `Db.sid2Id`, we need to handle the string created above in `Db.session2Sid`. Split the Sid string up as `x@_index:_id` and return the index.